ui/helpers/utils/util.js

```diff
@@ -631,3 +635,58 @@ export const getNetworkNameFromProviderType = (providerName) => {
 export const isAbleToExportAccount = (keyringType = '') => {
   return !keyringType.includes('Hardware') && !keyringType.includes('Snap');
 };
+
+/**
+ * Converts hex string to decimal value.
+ *
+ * @param {*} hex - string hex to convert to decimal
+ * @returns string decimal
+ */
+
+export const hexToDecimal = (hex) => parseInt(hex, 16);
+
+/**
+ * Converts decimal string to hex string
+ *
+ * @param {*} dec - string to convert to hex
+ * @returns string hex
+ */
+export const decimalToHex = (dec) => {
+  const decimalNumber = parseInt(dec, 10); // 10 is the base for decimal
+  return `0x${decimalNumber.toString(16)}`;
+};
```

Comment:
Can these existing functions be used?

https://github.com/MetaMask/metamask-extension/blob/develop/shared/modules/conversion.utils.ts#L166-L172

Reply:
Looks more convenient! the decimal to hex does not concatenate the '0x' but i could do that 🙏